documentation/sections/breaking_changes_argument_parsing_and_shebang_lines.md

```diff
@@ -0,0 +1,69 @@
+### Argument parsing and shebang lines
+
+#### Trailing `--`
+
+Previous versions of CoffeeScript required a `--` after the script to run, but this convention is now deprecated. The new standard is described in the output of `coffee -h`:
+
+``` bash
+> coffee -h
+
+Usage: coffee [options] [--] path/to/script.coffee [args]
+
+If called without options, `coffee` will run your script.
+
+
+```
```

Comment:
I actually screwed up the help text (see the most recent commit -- needed to change the help method to use `@rules.ruleList` instead of just `@rules`). I have also added tests in the most recent commit to ensure the help text will always display the banner and the flags.